examples/api-dotenv-request/app/Main.hs

```diff
@@ -0,0 +1,31 @@
+import Configuration.Dotenv (loadFile, defaultConfig)
+import System.Environment (lookupEnv)
+import Network.HTTP.Req
+import qualified Data.ByteString.Char8 as B
+import qualified Data.Text as T
+import Data.Aeson
+
+main :: IO ()
+main = do
+    loadFile defaultConfig
+    
+    --  Loading API URI from dotenv 
+    maybeApiEndpoint <- lookupEnv "API_URI"
+
+    case maybeApiEndpoint of
+        Just apiEndpoint -> do
+            -- Making the request
+            let apiEndpointURL = https (T.pack apiEndpoint) /: T.pack "posts" /: T.pack "1"
+            
+            --  Fake user agent
+            let userAgent = "Mozilla/5.0 (Linux; Android 5.0.1; Nokia 1100 LTE Build/GRK39F) AppleWebKit/536.14 (KHTML, like Gecko)  Chrome/55.0.3442.377 Mobile Safari/600.3"
```

Comment:
You don't need to fake the user agent like this. Just put an identifying string like "Haskell Network.HTTP.Req" or something